src/apiRESTTest/hibernateutil/HibernateUtil.java

```diff
@@ -0,0 +1,29 @@
+package apiRESTTest.hibernateutil;
+
+
+import org.apache.log4j.BasicConfigurator;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+
+	private static SessionFactory factory;
+
+	public SessionFactory config() {
```

Comment:
I would make this method private, then an additional public method `getSessionFactory?` would call config and return the `SessionFactory`; I would even rename `config` to `buildSessionFactory`

Reply:
Refactor done!